app/helpers/project_vulnerability_reports_helper.rb

```diff
@@ -1,15 +1,18 @@
 module ProjectVulnerabilityReportsHelper
   def pointer(pvr, pvs = false)
-    extra_css = ''
     if pvs
       score = pvr.vulnerability_score.to_f
-      percentage = ((score / max_score) * 84).round
```

Comment:
I think it's a magic number CSS fix. When I tried with 100, (actual percentage), the bar actually overextends. So we used an extra css variable so that whenever the percentage is around 100, we add the extra css to reach the end.